### PR TITLE
Gametime Slider

### PR DESCRIPTION
**Pull request by tend2infinity**
Solves issue #85 
It is in progress, I'll be able to complete this by tomorrow.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -22,8 +22,13 @@
     "bootstrap": "^4.5.2",
     "email-validator": "^2.0.4",
     "hover.css": "^2.3.2",
+<<<<<<< HEAD
```
Comment (apurv-1) [inline client/package.json:25]: Hey, you have a merge conflict here.

**Review by apurv-1 — COMMENTED** [review]
You have some merge conflicts on package.json

**Comment by adi-g15** [comment]
@tend2infinity Any further update on this, or are you stuck somewhere ?

**Comment by tend2infinity** [comment]
Nope, I was just a bit occupied since the last few days, I'll complete that today!

**Comment by fave77** [comment]
> Nope, I was just a bit occupied since the last few days, I'll complete that today!

Hi @tend2infinity , do you still plan to complete this PR?

**Pull request closed without merging**